tree-sitter-darklang/grammar.js

```diff
@@ -355,6 +357,34 @@ module.exports = grammar({
         ),
       ),
 
+    if_expression: $ =>
+      seq(
+        field("keyword_if", alias("if", $.keyword)),
+        field("condition", $.expression),
+        field("keyword_then", alias("then", $.keyword)),
+        choice(
+          seq(
+            $.indent,
+            repeat1(field("then_expression", $.expression)),
+            $.dedent,
+          ),
+          field("then_expression", $.paren_expression),
```

Comment:
it's neat that you needed a paren_expression here, but not in the equivalent else_expression below. This tree-sitter conflict stuff continues to be a mystery to me.